### ISSUE TITLE
Fetch tweets from CSV files

### ISSUE THREAD

**Issue by formulated** [digitalmethodsinitiative/dmi-tcat#390]
Hi. Thank you for your contribution to this programming. I really enjoy using TCAT.

One issue is about fetching tweets from csv files. I installed TCAT on AWS's Ubuntu. I do know any knowledge about php language. I have a twitter URL csv file.Thus, I need to use "part-csv.php." 

It is quite unclear how to use the fetch function. Even with several hours googling, I cannot find any way to solve it. The same issues would be applied to most FAQ information (e.g., search, user timelines, import, and etc.)

Do I have to change the source code? such as using "sudo nano parse-csv php", or just "php parse-csv.php climatechange-tweets.csv lookup-ids.txt." 

If the latter is correct,  How can I upload my csv file on the TCAT. 
Do I have to use AWS bucket to read csv file? or Can I use my local file? 
e.g., php parse-csv.php s3://.../url.csv, lookup-ids.txt 

In sum, without CS knowledge, it is hard to follow your instruction of FAQ

If you give a more clear explanation, such as a tutorial, more people can enjoy your effort!

**Comment by formulated** [comment]
Thank you. I solved the problem by myself!

**Comment by orangebacked** [comment]
Dear formulated, i'm stuck in the same problem, could you explain the solution !! It would be very useful!!
